<?xml version="1.0" ?>
<launch>

  <include file="$(find darknet_ros)/launch/darknet_ros.launch" >
   <arg name="yolo_weights_path" value="$(find sd_sample_pkg)/external_pkgs/darknet_ros/model"/>
   <arg name="yolo_config_path" value="$(find sd_sample_pkg)/external_pkgs/darknet_ros/model"/>
   <arg name="ros_param_file" value="$(find sd_sample_pkg)/external_pkgs/darknet_ros/config/ros.yaml"/>
   <arg name="network_param_file" value="$(find sd_sample_pkg)/external_pkgs/darknet_ros/config/yolov3-tiny.yaml"/>
   <arg name="image" value="/camera/image_raw"/>
  </include>
  <include file="$(find sd_sample_pkg)/launch/robot_simulation.launch" />


</launch>
